lib/puppet_x/sqlserver/sql_connection.rb

```diff
@@ -50,8 +50,8 @@ def get_connection_string(config)
           # SQL Server based authentication
           raise ArgumentError, _('admin_user must not be empty or nil') unless admin_user != ''
           raise ArgumentError, _('admin_pass must not be empty or nil') unless admin_pass != ''
-          params.store('User ID',  admin_user)
-          params.store('Password', admin_pass)
+          params.store('UID',  admin_user)
+          params.store('PWD', admin_pass)
```

Comment:
Are these changes specific to 2022? Or does is it backwards compatible?

Reply:
@chelnak these changes along with the updated Provider are all backwards compatible, at least as far back as SQL Server 2012